package.json

```diff
@@ -9,7 +9,7 @@
     },
     "scripts": {
       "test": "markdownlint ./cheatsheets/ --ignore node_modules",
-      "link-check" : "find  cheatsheets -name \\*.md -exec markdown-link-check -c mlc_config.json {} \\; 2> err && if [ -e err ] && grep -q  \"ERROR:\" err ; then grep \"ERROR:\" err exit 113  ; else echo -e \"All good\"; fi"
+      "link-check" : "find  cheatsheets -name \\*.md -exec markdown-link-check -c mlc_config.json {} \\; 2> err && if [ -e err ] && grep -q  \"ERROR:\" err ; then exit 113  ; else echo -e \"All good\"; fi"
```

Comment:
Why not simply go the `cheatsheets` directory, and run the md link check over the whole repo? And if we are setting the config file, I'd prefer we name it more descriptive `markdown-link-check_config.json` maybe?

Reply:
I forgot to put the config file name back, changed it to test the actions job. will update it to `markdown-link-check-config.json`

Reply:
As we are doing it in single command, change directory will be another section inthe command. Still it checks only the cheatsheets directory as we use `find cheetsheets`